<<<<<<< HEAD
#bloco else se o try não gerrar exceção
try:
    x = 10 / 2
except ZeroDivisionError as e:
    print(f"Erro de divisão por zero: {e}")
else:
    print(f"Resultado: {x}")
=======
#bloco else se o try não gerrar exceção
try:
    x = 10 / 2
except ZeroDivisionError as e:
    print(f"Erro de divisão por zero: {e}")
else:
    print(f"Resultado: {x}")
>>>>>>> 08b2136512fad3bd18765b823f223fa35b93a67d
